<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChattingClass</class>
 <widget class="QWidget" name="ChattingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>536</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>OPPOSans M</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Chatting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="messageDisplay">
       <property name="font">
        <font>
         <family>OPPOSans M</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="textBox">
         <property name="font">
          <font>
           <family>OPPOSans M</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="font">
          <font>
           <family>OPPOSans M</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
